changelog.md

```diff
@@ -0,0 +1,15 @@
+# v0.8.0
```

Comment:
I don't think so. IbPy2 is just the package name on PyPI, the version can be anything 0.x, 1.x, 2.x etc. that should be really independent of the package name. I went for 0.8 as 0.7.x was the last version I could find anywhere (PyPI, google code and GH)